packages/@okta/vuepress-site/docs/guides/build-sso-integration/specify-your-settings/openidconnect/settings.md

```diff
@@ -17,13 +17,13 @@
       * **Allow Access Token with implicit grant type**
   >**Note:** For SPA app integrations, the **Authorization Code** grant type always uses PKCE to verify the client. Also, the **Client acting on behalf of itself** grant type is not supported in OIN app integrations.
 * If you only want to support direct SSO to your application (so the integration isn't launched from the Okta End-User Dashboard), then:
-  1. Enter one or more **Login redirect URI** values where Okta sends the OAuth responses.
-  1. Set the **Login initiated by** drop-down box to **App Only**.
+  1. Enter one or more **Sign-in redirect URIs** values where Okta sends the OAuth responses.
+  1. Set the **Sign-in initiated by** drop-down box to **App Only**.
   1. Leave all of the remaining entries at their default values.
 
 * If you want to support launching your application from the Okta dashboard:
-  1. Enter one or more **Login redirect URI** values where Okta sends the OAuth responses.
-  2. (Optional). Enter the **Logout redirect URIs** where Okta redirects the browser after receiving the sign out request from the relying-party and terminating its end-user session. See [Single Logout](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_Single_Logout) or the [`/logout` API endpoint](/docs/reference/api/oidc/#logout).
+  1. Enter one or more **Sign-in redirect URIs** values where Okta sends the OAuth responses.
+  2. (Optional). Enter the **Sign-out redirect URIs** where Okta redirects the browser after receiving the sign out request from the relying-party and terminating its end-user session. See [Single Logout](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_Single_Logout) or the [`/logout` API endpoint](/docs/reference/api/oidc/#logout).
```

Comment:
```suggestion
  2. (Optional). Enter the **Sign-out redirect URIs** where Okta redirects the browser after you receive the sign-out request from the relying-party and terminate its end-user session. See [Single Logout](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Apps_Single_Logout) or the [`/logout` API endpoint](/docs/reference/api/oidc/#logout).
```

Reply:
Changed to:
> Enter the **Sign-out redirect URIs** where Okta redirects the browser after it receives the sign-out request from the relying-party and terminates the end-user's session. 
{Adding "you" would be ambiguous here - is it you the end-user, you the admin, or you the app?}